tools/AzureFunctionsJavaWorker.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Azure.Functions.JavaWorker</id>
+    <title>Microsoft Azure Functions Java Worker</title>
+    <tags>java azure-functions azure</tags>
+    <version>1.1.0-beta1-$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <projectUrl>https://github.com/Azure/azure-functions-java-worker</projectUrl>
+    <licenseUrl>https://github.com/Azure/azure-functions-java-worker/blob/master/LICENSE</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/Azure/azure-functions-cli/master/src/Azure.Functions.Cli/npm/assets/azure-functions-logo-color-raster.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Microsoft Azure Functions Java Worker</description>
+    <copyright>Copyright (c) Microsoft Corporation. All rights reserved.</copyright>
+    <contentFiles>
+      <files include="../azure-functions-java-worker/target/azure-functions-java-worker-1.0-SNAPSHOT.jar" buildAction="None" copyToOutput="true" flatten="false"/>
```

Comment:
So we need to update this (1.0-SNAPSHOT) for every new version (maybe to 1.0.0-beta-2 for the next release)?

Reply:
Let me try something to make this more generic.